sysboundary/sysboundary.cpp

```diff
@@ -385,6 +386,9 @@ bool SysBoundary::checkRefinement(dccrg::Dccrg<spatial_cell::SpatialCell, dccrg:
       }
    }
 
+   // For debugging
+   bool success = true;
```

Comment:
This and 410+414 below looks like it's not really needed?

Reply:
Right, I think I had originally rigged this to print every cell offending the boundary check, but that's basically superfluous esp. with the boundary check being mostly fluff now.